source/crud/read-operations/cursor.txt

```diff
@@ -7,10 +7,11 @@ Access Data From a Cursor
 Read operations that return multiple documents do not immediately return
 all values matching the query. Because a query can potentially match
 very large sets of documents, these operations rely upon an
-object called a **cursor** that fetches documents in batches to reduce
-both memory consumption and network bandwidth usage. Cursors
-are highly configurable and offer multiple interaction paradigms for
-different use cases.
+object called a `cursor
+<https://mongodb.github.io/node-mongodb-native/3.4/api/Cursor.html>`_
```

Comment:
I think you can use the `node-docs` role here:
https://github.com/mongodb/snooty-parser/blob/master/snooty/rstspec.toml#L530

Reply:
Yeah, I definitely need to update a lot of links with this.